org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/ApplicationProperties.java

```diff
@@ -358,4 +360,157 @@ public String description() {
+    }
+  }
+
+  public static class ScoutApplicationGzipExcludedPaths extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.excluded.paths";
+    }
+
+    @Override
+    public String description() {
+      return "Excluded paths for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedInflatePaths extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.inflate.paths";
+    }
+
+    @Override
+    public String description() {
+      return "Included inflate paths for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedMethods extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.methods";
+    }
+
+    @Override
+    public String description() {
+      return "Included methods for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedMimeTypes extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.mimeTypes";
+    }
+
+    @Override
+    public String description() {
+      return "Included MIME types for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipIncludedPaths extends AbstractStringListConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.included.paths";
+    }
+
+    @Override
+    public String description() {
+      return "Included paths for " + GzipHandler.class.getName() + " (respected only if at least one is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipMinSize extends AbstractPositiveIntegerConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.minSize";
+    }
+
+    @Override
+    public String description() {
+      return "Minimum size to trigger compression for " + GzipHandler.class.getName() + " (respected only if at value is set otherwise Jetty default is used).";
+    }
+  }
+
+  public static class ScoutApplicationGzipSyncFlush extends AbstractBooleanConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.syncFlush";
+    }
+
+    @Override
+    public String description() {
+      return "Enables sync flush for " + GzipHandler.class.getName() + ".";
```

Comment:
Add "Default value is " + getDefaultValue();

Reply:
https://github.com/eclipse-scout/scout.rt/compare/ddfce395c1ce7e5e0bb3ec14ca3342013320f395..3e80c6ba06d95d99f3212f4e027768c687cc2451